extensions/xcode/package.json

```diff
@@ -239,6 +239,21 @@
       "description": "Removes the ~/Library/Developer/Xcode/DerivedData directory.",
       "mode": "no-view"
     },
+    {
+      "name": "open-url-in-simulator.command",
+      "title": "Open URL in Simulator",
+      "subtitle": "Xcode",
+      "description": "Open the given URL in the currently booted simualtor.",
+      "mode": "no-view",
+      "arguments": [
+        {
+          "name": "URL",
```

Comment:
The `name` must match with the property name inside the `LaunchProps` argument. So changing `URL` to `url` should do the job.